src/classes/Contacts.cls

```diff
@@ -50,10 +61,133 @@ public inherited sharing class Contacts extends fflib_SObjects {
         synchronizeNamingExclusions();
     }
 
+    public void onBeforeInsertAddrConTDTM() {
```

Comment:
What are the TDMT() methods here? I think these should have block comments to make usage clear since they're clearly different than the other onModeAction method.